contracts/adder/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "adder"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
+edition = "2021"
+
+[dependencies]
+anyhow = "*"
+ink = { version = "4.0.0-beta", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
+
+[lib]
+name = "adder"
+path = "lib.rs"
+crate-type = [
+	# Used for normal contract Wasm blobs.
+	"cdylib",
```

Comment:
Looks like formatting here is a bit different.

Reply:
Merged code, same.